docker-compose.yaml

```diff
@@ -58,6 +35,47 @@ services:
     networks:
       - atoma-network
 
+  vllm:
+    container_name: chat-completions
+    profiles: [chat_completions_vllm]
+    image: vllm/vllm-openai:v0.6.3
```

Comment:
why not `latest` ? Manually having to track each version is not ideal, and I believe that any potential changes to openai api on vLLM will be prior advertised

Reply:
I thought about that, and it's another possible way. But see the following reasons against using `latest`:
- https://vsupalov.com/docker-latest-tag/

For me, I like having the peace of mind to know that **atoma-node won't break from one day to another with the same git commit**. Which is not the case if we rely on `latest` (a bit the same philosophy to using `Cargo.lock`, if we specify the version to of the images, we know for a fact that everyone that runs `atoma-node` on the same git commit, they will have the same build running).

Reply:
Sounds good, I am happy with it